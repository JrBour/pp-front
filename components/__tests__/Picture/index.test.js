import { shallowMount } from '@vue/test-utils'
import Picture from '@/components/Picture'

describe('Picture', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallowMount(Picture, {
      propsData: {
        media: 'lorem-ipsum',
      },
    })
  })

  describe('snapshot', () => {
    it('renders properly', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
