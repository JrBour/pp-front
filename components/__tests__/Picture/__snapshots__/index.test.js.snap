// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture snapshot renders properly 1`] = `<div style="background-image: url(undefined/media/lorem-ipsum);"></div>`;
